### ISSUE TITLE
Unnecessary complexity

### ISSUE THREAD

**Issue by igoyak** [YoungxHelsinki/automated-scripts#5]
https://github.com/YoungxHelsinki/automated-scripts/blob/3b8fd3813c73b96826b5693343e496797a13f9ee/Aalto_computer_status_checker/paniikki_jupyter_launcher.py#L137-L159

This whole block can be replaced with one line. You can sort based on multiple elements. 
Example code:

```
from random import randint

class Node():
	def __init__(self):
		self.users = randint(5, 7)
		self.load = randint(0, 3)
	def __str__(self):
		return 'users: %s, load: %s' % (self.users, self.load)

nodes = [Node() for _ in range(10)]
for n in nodes:
	print(n)

print()
nodes.sort(key=lambda n: (n.users, n.load)) # This is the line you want

for n in nodes:
	print(n)
```

**Comment by SuperShinyEyes** [comment]
Aha, so you can sort by multiple columns. Nice!

**Comment by SuperShinyEyes** [comment]
Fixed @ 4452a1cb027babf6e11010cd0225d45d437f7785
